src/plugins/discord/mirror.test.js

```diff
@@ -33,11 +35,15 @@ describe("plugins/discord/mirror", () => {
 
     function testMessages(channelId) {
       if (channelId === activeChannelId) {
-        const authorId = "1";
         const reactionEmoji = [customEmoji()];
         return [
-          testMessage(activeMessageid, channelId, authorId, reactionEmoji),
-          testMessage("2", channelId, authorId, []),
+          testMessage(
+            activeMessageid,
+            channelId,
+            activeAuthorId,
+            reactionEmoji
+          ),
+          testMessage("2", channelId, activeAuthorId, []),
```

Comment:
I recognize this hasn't changed since last time -- but it seems kind of weird that it returns message id 34 and then message id 2? Wouldn't the message ids be increasing in practice? Does this not matter?